.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+fail_fast: true
+minimum_pre_commit_version: '0'
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: check-added-large-files
+    -   id: check-case-conflict
+    -   id: check-json
+    # -   id: check-yaml # skip for now
```

Comment:
Any reason? I am not a fan of commented code, as it does not serve any purpose. It can be replace via human readable documentation though.